src/brainbrowser/volume-viewer/modules/loading.js

```diff
@@ -508,7 +509,7 @@ BrainBrowser.VolumeViewer.modules.loading = function(viewer) {
       }
 
       Array.prototype.forEach.call(template, function(node) {
-        if (node.nodeType === 1) {
+        if (node.nodeType === 1 && volume_description.show_volume !== false) {
```

Comment:
10 lines up is `if` skipping this when no `template` is provided (note that `template_options` defaults to `{}`).

Reply:
see reply to previous comment